magma/__init__.py

```diff
@@ -105,3 +105,11 @@ def set_mantle_target(t):
 from magma.compile import MagmaCompileException
 from magma.linking import link_module, link_default_module, clear_link_info
 import magma.math
+
+################################################################################
+# BEGIN ALIASES
+################################################################################
+Reg = Register
+################################################################################
+# END ALIASES
+################################################################################
```

Comment:
Should we consider providing this in magma.primitives.register and importing it here? This would allow internal code (e.g. other primitives) to reference it without using the top level import

Reply:
yeah I could go either way on it. I do kind of like the isolation of having a set of aliases only for external users (i.e. through __init__), which seems like a different case than desiring it internally.

If we see a reason to use Reg internally then I can move it there, otherwise I'd argue it should be in `__init__.py`